src/server/tests/connectivity_test.go

```diff
@@ -2,6 +2,7 @@ package server_test
 
```

Comment:
Improper usage of a mock. Mocks are for replacing the real implementations of components of the system. We have a component called "Client" and the naming here implies you're mocking that object. Please explain what the issue with separate utility functions is, because I believe that's the cleanest solution. Splitting them into separate files is good.

Reply:
Is TestClient alright?

Reply:
That's better.